packages/salesforcedx-vscode-automation-tests/test/specs/sfdxProjectJson.e2e.ts

```diff
@@ -74,9 +72,8 @@ const createSfdxProjectJsonWithAllFields = async (testSetup: TestSetup): Promise
     '\t"sourceBehaviorOptions": ["decomposeCustomLabelsBeta", "decomposePermissionSetBeta", "decomposeWorkflowBeta", "decomposeSharingRulesBeta"]',
     '}'
   ].join('\n');
-  await openFile(path.join(testSetup.projectFolderPath!, 'sfdx-project.json'));
-  const textEditor = await getTextEditor(workbench, 'sfdx-project.json');
-  await textEditor.setText(sfdxConfig);
-  await textEditor.save();
+
+  const sfdxProjectPath = path.join(testSetup.projectFolderPath!, 'sfdx-project.json');
+  await fs.writeFile(sfdxProjectPath, sfdxConfig, 'utf8');
```

Comment:
I think we should have an utility in redhat repo to do this and use it instead. Wdyt?